doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -360,11 +360,13 @@ Thread samples
 Matter samples
 --------------
 
+* Enabled Matter shell commands for all build types except ``release``.
```

Comment:
```suggestion
* Enabled Matter shell commands for all build types except ``release`` in all Matter samples.
```
Just a little bit of additional clarity.

Though CLI is only mentioned in the light switch sample. Should this be only in reference to it?